src/index.d.ts

```diff
@@ -633,6 +1310,9 @@ declare namespace admin.auth {
     ): Promise<admin.auth.AuthProviderConfig>;
   }
 
+  /**
```

Comment:
Perhaps this empty block can be removed?

Reply:
You bet Hiranya -- staged at https://firebase.devsite.corp.google.com/docs/reference/admin/node/admin.auth.UserRecord.

As you noticed, I should clarify that the Auth methods commenting will come in a separate following pull request.  Thanks!